### ISSUE TITLE
Vietnamese support for mono fonts

### ISSUE THREAD

**Issue by joulev** [mozilla/Fira#230]
```
ă â đ ê ô ơ ư
ắ ằ ẳ ẵ ặ
ấ ầ ẩ ẫ ậ
ế ề ể ễ ệ
ố ồ ổ ỗ ộ
ứ ừ ử ữ ự
(and Ă Â Đ etc.)
```

All of the above letters are included in the sans version

![image](https://user-images.githubusercontent.com/44609036/64842524-a401d780-d62c-11e9-986b-7c991f4eba73.png)

However they seems to be excluded in the mono version

![image](https://user-images.githubusercontent.com/44609036/64842566-c0057900-d62c-11e9-87d9-1226859a77b3.png)

I think those letters should be supported in mono as well.

All screenshots are taken from http://mozilla.github.io/Fira/ at the time of this issue.

---

I have read #19, #172 and #223, but currently the font on the official site itself doesn't work, so I don't think the font in the releases would work.

**Comment by dinhquochan** [comment]
Any updates for this?

**Comment by j-f1** [comment]
You’d likely have more luck asking for this over at https://github.com/tonsky/FiraCode.

**Comment by dinhquochan** [comment]
> You’d likely have more luck asking for this over at https://github.com/tonsky/FiraCode.

Yes, I was asked in 2016

![image](https://user-images.githubusercontent.com/9979458/79406925-774adf80-7fc2-11ea-9a39-0308402b2506.png)


Ref [FiraCode#185](https://github.com/tonsky/FiraCode/issues/185)

**Comment by j-f1** [comment]
Seems like it was recently reopened

**Comment by joulev** [comment]
@j-f1 Yes, but I believe this is the more appropriate place for the issue. Even if the issue in https://github.com/tonsky/FiraCode/issues/185 is resolved, normal Fira Mono users still can't write in Vietnamese.

**Comment by fitojb** [comment]
Fira hasn’t been developed since 2016, when Mozilla pulled out. The successor project is a reclamation by its authors called FiraGO. Any new request should be posted to their repo instead of here. Why this repo hasn’t been archived is beyond me.
